controller/src/main/java/org/jboss/as/controller/interfaces/LoopbackAddressInterfaceCriteria.java

```diff
@@ -61,18 +61,19 @@ public synchronized InetAddress getAddress() throws UnknownHostException {
         return this.resolved;
     }
 
-        /**
+    /**
      * {@inheritDoc}
      *
      * @return <code>{@link #getAddress()}()</code> if {@link NetworkInterface#isLoopback()} is true, null otherwise.
      */
     @Override
     protected InetAddress isAcceptable(NetworkInterface networkInterface, InetAddress address) throws SocketException {
```

Comment:
This should no longer throw a SocketException, as this is now handled.

Reply:
@pferraro Well spotted! Opened https://github.com/wildfly/wildfly-core/pull/6398 as a fixup.

Reply:
I missed this because it was NOT documented in the Javadoc...